src/main/java/io/chrislowe/discordle/Main.java

```diff
@@ -132,6 +135,14 @@ public void registerCommands(GatewayDiscordClient gateway) {
                 .createGlobalApplicationCommand(applicationId, userCommandRequest)
                 .subscribe();
 
+        ApplicationCommandRequest recentCommandRequest = ApplicationCommandRequest.builder()
+            .name("recent")
+            .description("View recent games!")
+            .build();
+        gateway.getRestClient().getApplicationService()
+            .createGuildApplicationCommand(applicationId, 190983259490877440L, recentCommandRequest)
```

Comment:
I think we want to make this a global app command

Reply:
LOL yes we do

Reply:
Done